src/components/P5AP_PhotoCard/photocard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './PhotoCard.scss'; 
+import idea from '../../assets/icons/idea.svg'
```

Comment:
You should really think about using an icon library.

better to name this ideaIcon and briefcaseIcon. The way this is named is unreadable